pipelines/src/pipelines/utils/trigger_pipeline.py

```diff
@@ -67,6 +69,10 @@ def trigger_pipeline(
         network=network,
     )
 
+    if wait.lower() == "true":
+        pl.wait()
+    elif wait.lower() != "false":
+        raise ValueError("wait variable must be 'true', 'false', or not set.")
```

Comment:
I would suggest to move this logic to the `if __name__ == "__main__":` block, and make the `wait` argument in `trigger_pipeline` a `bool`

```python
def trigger_pipeline(
    template_path: str,
    display_name: str,
    wait: bool = False,
):
...

if __name__ == "__main__":
    ...
    parser.add_argument(
        "--wait",
        help="Wait for the pipeline to finish running",
        type=str,
    )
    
    if args.wait.lower() == "true":
        wait = True
    elif args.wait.lower() != "false":
        raise ValueError("wait variable must be 'true' or 'false'")
    wait = False
    
    trigger_pipeline(
        template_path=args.template_path,
        display_name=args.display_name,
        wait=wait,
    )
```